data/data/rhcos-amd64.json

```diff
@@ -1,135 +1,135 @@
+        "image": "rhcos-44-81-202003062006-0-gcp-x86-64",
+        "url": "https://storage.googleapis.com/rhcos/rhcos/rhcos-44-81-202003062006-0-gcp-x86-64.tar.gz"
     },
     "images": {
         "aws": {
-            "path": "rhcos-44.81.202002241126-0-aws.x86_64.vmdk.gz",
-            "sha256": "2a6a1c5ba0dd6819c1df11e210e1752dead6e92bfc8dc9f88ff4bdfe826589af",
-            "size": 866196706,
-            "uncompressed-sha256": "e480f3cd790f81c94f166b8706ee7aba51ef474c6a843b36e54c597563d19436",
-            "uncompressed-size": 884048896
+            "path": "rhcos-44.81.202003062006-0-aws.x86_64.vmdk.gz",
+            "sha256": "3c96142f5f9c2362864abb362f99256eb6e3b5ca01740da39e7ad251f2a42ff6",
+            "size": 862543887,
+            "uncompressed-sha256": "f5a93a7616835785a74d54c7ef123a2ef634124e6f6c4b743b310e5f6b2c5404",
+            "uncompressed-size": 880251904
         },
         "azure": {
-            "path": "rhcos-44.81.202002241126-0-azure.x86_64.vhd.gz",
-            "sha256": "d7b3fcc3cce5c27bd5b0b844d014efdf2f8a1979b48c24d8c701b2fb4fd16a36",
-            "size": 866305735,
-            "uncompressed-sha256": "83e42fe90c1e6a1d742c995dc0354e41d739c5bb4c488f8f621962570a6fb2af",
+            "path": "rhcos-44.81.202003062006-0-azure.x86_64.vhd.gz",
+            "sha256": "45f5838fea89db9757be9a02e68515c6dfc9e8ebaf39f65dc90e1219261ede8f",
+            "size": 862671452,
+            "uncompressed-sha256": "8b6cb9a49697c1a1ecc1a121f4d4b1add397cc4f7c1ea67d49de6384b7e07da5",
             "uncompressed-size": 17179869696
         },
         "gcp": {
-            "path": "rhcos-44.81.202002241126-0-gcp.x86_64.tar.gz",
-            "sha256": "c777d94442f99d831030fc408ace25b29f8ff0913ef10923384b9a4e767b4029",
-            "size": 851548572
+            "path": "rhcos-44.81.202003062006-0-gcp.x86_64.tar.gz",
+            "sha256": "679de9281b3671100f520dd82d00bdcb877924dbeb4df7c3eb912374377b5898",
+            "size": 847864998
         },
         "initramfs": {
-            "path": "rhcos-44.81.202002241126-0-installer-initramfs.x86_64.img",
-            "sha256": "91b6cb390afea317b3bc7eff62e330465613a1aaf7a3595545ad98c564deb815"
+            "path": "rhcos-44.81.202003062006-0-installer-initramfs.x86_64.img",
+            "sha256": "88296036a637a650ad6bdc63f61618227f587b990a41f61075b69d385c688975"
         },
         "iso": {
-            "path": "rhcos-44.81.202002241126-0-installer.x86_64.iso",
-            "sha256": "1939f2ef942a889ec7df546efd92769b8a5d58190436df45b484bc051b2503f8"
+            "path": "rhcos-44.81.202003062006-0-installer.x86_64.iso",
+            "sha256": "bdd4246b764068d42c86735be539cfa750c4083a9b6baaff434dae90e43772f2"
         },
         "kernel": {
-            "path": "rhcos-44.81.202002241126-0-installer-kernel-x86_64",
+            "path": "rhcos-44.81.202003062006-0-installer-kernel-x86_64",
             "sha256": "4d7f7b0a631a8f3fd34c9d39e7a037655871f05d503af240e7647a5f4e6490c9"
```

Comment:
kernal sha also didn't change :P

Reply:
This doesn't look to have changed in content though I'll ask for verification.

Reply:
Looks like the sha hasn't change since bd6291c. I'll open a bug up against the build pipeline to ensure that it starts getting updated once more. Likely the same thing as https://github.com/openshift/installer/pull/3257#discussion_r389210930

Reply:
```
$ sha256sum rhcos-44.81.202003062006-0-installer-kernel-x86_64 
4d7f7b0a631a8f3fd34c9d39e7a037655871f05d503af240e7647a5f4e6490c9  rhcos-44.81.202003062006-0-installer-kernel-x86_64
```

Reply:
RHCOS *is* RHEL, and in RHEL the model is we deliver pre-tested binaries; we're not rebuilding things from source very often.  This kernel is literally the same `vmlinuz` from the RHEL kernel RPM - it's completely normal for it not to change between RHCOS builds.